lib/internal/errors.js

```diff
@@ -1365,8 +1365,9 @@ E('ERR_WORKER_NOT_RUNNING', 'Worker instance not running', Error);
 E('ERR_WORKER_OUT_OF_MEMORY', 'Worker terminated due to reaching memory limit',
   Error);
 E('ERR_WORKER_PATH',
-  'The worker script filename must be an absolute path or a relative ' +
-  'path starting with \'./\' or \'../\'. Received "%s"',
+  'The worker script or module filename must be an absolute path or a ' +
+  'relative path starting with \'./\' or \'../\'. If you are trying to pass ' +
+  'a file:// URL, you should wrap it around `new URL`. Received "%s"',
```

Comment:
Could we maybe only show the additional text i the path starts with `file://`? I think that would be the most helpful variant